frontend/src/util/helper.js

```diff
@@ -182,14 +182,17 @@ export const sortContentList = (workspaceContents, lang) => {
 export const toggleFavicon = (hasUnreadNotification, hasUnreadMention) => {
   const originalHrefAttribute = 'originalHrefAttribute'
 
-  document.getElementsByClassName('tracim__favicon').forEach(favicon => {
+  const favicons = document.getElementsByClassName('tracim__favicon')
+  for (let i = 0; i < favicons.length; i++) {
+    const favicon = favicons[i]
+
```

Comment:
I think this piece of code could be written like this (not tested). This comment is just for knowledge, I don't have a strong preference.

```suggestion
  const favicons = Array.from(document.getElementsByClassName('tracim__favicon')).forEach(favicon => {
```

Reply:
I do prefer this version